hoomd/hpmc/integrate.py

```diff
@@ -748,14 +747,20 @@ class Polyhedron(_HPMCIntegrator):
 
         a (float): Default maximum size of rotation trial moves.
 
-        translation_move_probability (float): Fraction of moves that are translation
-            moves.
+        translation_move_probability (float): Fraction of moves that are
+            translation moves.
 
         nselect (int): Number of trial moves to perform per particle per
             timestep.
 
-    Note:
+    Perform hard particle Monte Carlo of general polyhedra defined by their
+    vertices and faces (see `shape`). `Polyhedron` supports triangle meshes and
+    spheres only. The mesh must be free of self-intersections.
+
+    See Also:
+        Use `ConvexPolyhedron` for faster performance with convex polyhedra.
```

Comment:
Did we ever decide if we should encourage users to just use the `ConvexSpheropolyhedron` class? If I remember correctly it is the same under the hood when the sweep radius = 0.

Reply:
Yes, the overlap check is identical. ConvexSpheropolyhedron implements a few methods differently, but I think these are strict superset of ConvexPolyhedron when `N_verts > 2` and `sweep_radius=0`.  We can deal with this removal in a later PR: #798